### PR TITLE
CLDC-3680: Copy changes lettings property information questions

### PR DESCRIPTION
**Pull request by Dinssa**

### CODE REVIEW

File: app/models/form/lettings/subsections/property_information.rb [hunk 1]
```diff
@@ -10,8 +10,8 @@ def pages
     @pages ||= [
       uprn_questions,
       Form::Lettings::Pages::PropertyLocalAuthority.new(nil, nil, self),
-      Form::Lettings::Pages::MinRentValueCheck.new("local_authority_min_rent_value_check", nil, self, check_answers_card_number: nil),
-      Form::Lettings::Pages::MaxRentValueCheck.new("local_authority_max_rent_value_check", nil, self, check_answers_card_number: nil),
+      Form::Lettings::Pages::MinRentValueCheck.new("local_authority_min_rent_value_check", nil, self, check_answers_card_number: nil), # Soft validation
+      Form::Lettings::Pages::MaxRentValueCheck.new("local_authority_max_rent_value_check", nil, self, check_answers_card_number: nil), # Soft validation
```
Comment (kosiakkatrina) [inline app/models/form/lettings/subsections/property_information.rb]: Did you want to leave the comments here?

File: config/locales/forms/2023/sales/soft_validations.en.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ en:
             not_joint_purchase:
               title_text: "You told us the buyer’s savings were %{savings}."
               informative_text: "This is higher than we would expect."
-          deposit_value_check::
+          deposit_value_check:
```
Comment (kosiakkatrina) [inline config/locales/forms/2023/sales/soft_validations.en.yml]: nice

File: spec/requests/duplicate_logs_controller_spec.rb [hunk 3]
```diff
@@ -66,7 +66,6 @@
             it "displays check your answers for each log with correct questions" do
               expect(page).to have_content("Q5 - Tenancy start date", count: 3)
               expect(page).to have_content("Q7 - Tenant code", count: 3)
-              expect(page).to have_content("Q12 - Postcode", count: 3)
```
Comment (kosiakkatrina) [inline spec/requests/duplicate_logs_controller_spec.rb:69]: Why does this no longer show up?

Comment (Dinssa) [reply spec/requests/duplicate_logs_controller_spec.rb:69]: It's a question that was pre-2023, I've removed the page and question

File: app/models/form/lettings/subsections/property_information.rb (hunk 1 above)
Comment (Dinssa) [reply app/models/form/lettings/subsections/property_information.rb]: I thought I'd mark the questions I left out for the next person who works on the soft validations, I can remove if its not helpful.

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**